package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sparqljson-parse",
-  "version": "2.0.1",
+  "version": "2.1.0",
```

Comment:
This is not needed, I'll do this myself upon the next release. (as well as the changelog)

Reply:
It's fixed. I made a mistake with the rebases.